depend/secp256k1/src/ext.c

```diff
@@ -24,7 +24,7 @@ int secp256k1_ecdh_raw(const secp256k1_context* ctx, unsigned char *result, cons
 		ret = 0;
 	else
 	{
-		secp256k1_ecmult_const(&res, &pt, &s);
+		secp256k1_ecmult_const(&res, &pt, &s, 256);
```

Comment:
this needs to be verified

Reply:
Can you elaborate?

Reply:
ext.c is not present in upstream (our extention), so we need to make sure 256 is the right size here

Reply:
/cc @svyatonik PTAL?

Reply:
Not sure, if I am the right person to ask, but imo that's the correct change :)